package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zigbee-adapter",
```

Comment:
@dhylands Can you update this PR so I don't have to submit a new one, please? Could you just add the following line after the `name` field:

```javascript
"display_name": "Zigbee",
```